### DISCUSSION TITLE
Grüße vom Stammtisch Braunschweig

### DISCUSSION THREAD

**Discussion by MonkeybreadSoftware** [fmIDE/fmIDE#15] [General]
Tolles Werkzeug!

**Comment by mrwatson-de** [comment]
Danke / Thank you Christian!

It's going to be fun using and developing fmIDE into the future!

It was a great session yesterday - we went into great detail quite late into the evening, and some good stuff came out of it - I've already added a new [`Insert from URL` fmIDE Action](https://github.com/fmIDE/fmIDE/wiki/fmIDE-Actions#insert-from-url) into the [latest release](https://github.com/fmIDE/fmIDE/releases)!

**Reply by marcel-more** [reply]
Really great stuff! Thanks for presenting!
It's inspiring to thing about what possibilities are inside your magic box 😀👍
